perun-ldapc/src/main/resources/perun-ldapc.xml

```diff
@@ -624,12 +624,20 @@ http://www.springframework.org/schema/aop http://www.springframework.org/schema/
 							<property name="name" value="userCertDNs" />
 							<property name="namespace" value="urn:perun:user:attribute-def:virt" />
 							<property name="valueTransformer">
-								<bean class="cz.metacentrum.perun.ldapc.beans.RegexpValueTransformer">
-									<property name="replaceList">
+								<bean class="cz.metacentrum.perun.ldapc.beans.CompositeValueTransformer">
+									<property name="transformerList">
 										<list>
-											<bean class="cz.metacentrum.perun.ldapc.beans.RegexpSubst">
-												<property name="find" value="^[0-9]+[:]" />
-												<property name="replace" value="" />
+											<bean class="cz.metacentrum.perun.ldapc.beans.KeysetValueTransformer" />
+											<bean class="cz.metacentrum.perun.ldapc.beans.RegexpValueTransformer">
+												<property name="massTransformationPreferred" value="false" />
```

Comment:
Based on code/usage, ``RegexpValueTransformer`` doesn`t make sense without this to be ``false``, since regex is applied to each single value only. Mass method is inherited from ``ValueTransformerBase``, where no change is applied. How about making it default to ``false`` in the code? Also we could override mass method and simply do, what is done in ValueExtractors or anywhere else, where we collect results of changes applied.

Reply:
I have implemented the sensible settings for RegexpVAlueTransformer and removed the setting from spring context.